suse.sh

```diff
@@ -19,11 +19,7 @@
 verify_enable_platform() {
   feature="$1"
   if [ "$OS_MAJOR" = "6" ]; then
-    if [ "${feature}" = "GPS_ENABLED" ]; then
-      echo "false"
-    else
-      verify_gcc_enable "${feature}"
-    fi
+    verify_gcc_enable "${feature}"
   else
     verify_gcc_enable "${feature}"
   fi
```

Comment:
the outer conditional could be removed now, too

Reply:
5c5de257a0